charts/renovate/values.yaml

```
@@ -71,3 +71,21 @@ resources:
   #  memory: 128Mi
 
 envFrom: []
+
+redis:
+  # Configuration for a Redis subchart. Additional documenation at
+  # https://github.com/bitnami/charts/tree/master/bitnami/redis.
```

This is a super small nit, but the full stop here breaks this link for me

```suggestion
  # https://github.com/bitnami/charts/tree/master/bitnami/redis
```